src/js/services/forms.js

```diff
@@ -113,10 +113,11 @@ export default App.extend({
   fetchForm() {
     const channel = this.getChannel();
 
-    return Promise.resolve(Radio.request('entities', 'fetch:forms:definition', this.form.id))
+    return Promise.resolve(Radio.request('entities', 'fetch:forms:definition', this.form.id), get(this.action, 'id'))
```

Comment:
```suggestion
    return Promise.resolve(Radio.request('entities', 'fetch:forms:definition', this.form.id, get(this.action, 'id')))
```

Reply:
😬 ha.  apparently this is never used for action..at least in the tests